R/rules-other.R

```diff
@@ -26,26 +26,71 @@ add_brackets_in_pipe_one <- function(pd, pos) {
 #' @param indent_by The amont of spaces used to indent an expression in curly
 #'   braces. Used for unindention.
 wrap_if_else_multi_line_in_curly <- function(pd, indent_by = 2) {
-  if (pd$token[1] == "IF" &&
-    pd_is_multi_line(pd) &&
-    !is_curly_expr(pd$child[[5]])) {
-    pd$lag_newlines[5] <- 0L
+  if (is_cond_expr(pd)) {
+    pd <- wrap_if_else_multi_line_in_curly_since_is_if_expr(pd, indent_by)
+  }
+    pd
+}
+
+
+wrap_if_else_multi_line_in_curly_since_is_if_expr <- function(pd, indent_by = 2) {
```

Comment:
I didn't understand the function name when skimming through it. Maybe extract `wrap_if_multi_line_in_curly()` and `wrap_else_multi_line_in_curly()`and then write

```r
wrap_if_else_multi_line_in_curly <- function(pd, indent_by = 2) {
  if (is_cond_expr(pd)) {
    pd <- wrap_if_multi_line_in_curly(pd, indent_by)
    pd <- wrap_else_multi_line_in_curly(pd, indent_by)
  }
  pd
}
```

Reply:
Yes, that makes sense. It's just that I thought it was simple but it turned out to be quite complicated with all the edge cases... -.-